app/src/main/res/layout/fragment_login.xml

```diff
@@ -88,12 +88,14 @@
             android:layout_marginEnd="8dp"
             app:layout_constraintEnd_toEndOf="parent"
             android:layout_marginBottom="8dp"
-            app:layout_constraintBottom_toBottomOf="parent"/>
+            app:layout_constraintBottom_toBottomOf="parent"
+            app:layout_constraintHorizontal_bias="0.498"
+            app:layout_constraintVertical_bias="0.259"/>
 
     <androidx.constraintlayout.widget.Guideline
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
             android:id="@+id/guidelineMiddle"
             android:orientation="horizontal"
-            app:layout_constraintGuide_end="303dp"/>
+            app:layout_constraintGuide_end="410dp"/>
```

Comment:
je lepsi pouzivat `app:layout_constraintGuide_percent="0.XY"`  coz jsou procenta a taky staci zaokrouhlit na 1-2 desetinnych mist

Reply:
aha , ok - myslím, že mi to vygenerovalo AS samo pri posunovani

Reply:
hodnota by měla byt upravena + přidaný scrollView